libs/shared/ui/src/lib/components/inputs/input-radio-box/input-radio-box.tsx

```diff
@@ -21,7 +21,7 @@ export function InputRadioBox(props: InputRadioBoxProps) {
           onChange(value)
         }, 1)
       }}
-      className={`p-4 rounded border mb-2 transition transition-all ${
+      className={`p-4 rounded border mb-2 cursor-pointer transition-all ${
```

Comment:
I did not add the cursor pointer on purpose. Usually, a radio type button does not have a cursor on it. I prefer without it personally because once it's clicked, you still have the hand cursor while you can't really interact with the box anymore, and also because in these kind of boxes you can have other widget : links, input, select and to have the hand cursor for all of them is not the best imo. But as you prefer!

https://user-images.githubusercontent.com/6163954/217231078-9df6f8e6-56fc-46c0-9647-1ad105c75a2f.mp4